pkg/provisioner/controller.go

```diff
@@ -380,7 +439,18 @@ func (c *obcController) handleProvisionClaim(key string, obc *v1alpha1.ObjectBuc
 		return fmt.Errorf("error getting reference to OBC: %v", err)
 	}
 
-	// create Secret and ConfigMap
+	// Create auth Secret and bucket info ConfigMap
```

Comment:
`setBasicInfo` is called [here](https://github.com/kube-object-storage/lib-bucket-provisioner/pull/200/files#diff-1e346e3430c7a1bec9a1fde398367f33b921671c6c3ced081ad42e4d8e345522R400). What is the reason for also calling it in the [`defer`](https://github.com/kube-object-storage/lib-bucket-provisioner/pull/200/files#diff-1e346e3430c7a1bec9a1fde398367f33b921671c6c3ced081ad42e4d8e345522R400)? Is this to handle cases of the provisioner unsetting some of the ob fields?  

BTW, I know this pr does not change this code, but I noticed it in my review.

Reply:
The ob from the provisioner can come back without info about the storage class, but the cleanup 'Delete' will need that info. Basically, just fill out the known info more to eliminate problems cleaning up caused by lack of that info.